server/routes/post.routes.js

```diff
@@ -0,0 +1,15 @@
+const validateToken = require('../middlewares/validateToken.js')
+
+module.exports = app => {
+    const posts = require('../controllers/post.controller.js')
+
+    var router = require('express').Router()
+
+    router.post('/', validateToken, posts.insertPosts)
+    router.get('/', validateToken, posts.getUserPosts)
+    router.get('/all', posts.getAllPosts)
+    router.delete('/', validateToken, posts.deleteOnePost)
+    router.delete('/all', validateToken, posts.deleteAllPost)
+
+    app.use('/api/posts', router)

```

Comment:
No router for getting one specific post detail?
And I think it's better to use /{id} for delete one post